src/main.cpp

```diff
@@ -16,6 +16,7 @@ long int val; // variable to read the value from the analog pin
 void setup()
 {
   Serial.begin(115200);
+  radio.beginSerial(115200);
```

Comment:
It seems that `Serial` is not used anywhere, right? We should remove it.

Reply:
Removed `Serial.begin`.